app/Http/routes.php

```diff
@@ -384,18 +384,6 @@ function ($event_id) {
                 'uses' => 'EventTicketsController@postUpdateTicketsOrder',
             ]);
 
-            /*
-             * Ticket questions
-             */
-            Route::get('{event_id}/tickets/questions', [
-                'as'   => 'showTicketQuestions',
-                'uses' => 'EventTicketQuestionsController@showQuestions',
-            ]);
-            Route::post('{event_id}/tickets/questions/create', [
-                'as'   => 'postCreateQuestion',
-                'uses' => 'EventTicketQuestionsController@postCreateQuestion',
-            ]);
-
```

Comment:
@publicarray why did you remove these routes

Reply:
Unfortunately [EventTicketQuestionsController doesn't exist](https://github.com/Attendize/Attendize/search?q=EventTicketQuestionsController&unscoped_q=EventTicketQuestionsController) yet and it fixes https://github.com/Attendize/Attendize/issues/327.